README.md

```diff
@@ -47,6 +61,36 @@ To move a window, hold your `moveModifiers` down, then click and drag a window.
 
 To resize a window, hold your `resizeModifiers` down, then click and drag a window.
 
+### Fancy zones
+
+Fancy zones module is inspired by [Microsoft's FancyZones](https://docs.microsoft.com/en-gb/windows/powertoys/fancyzones) and aims to provide a similar way to interact with windows in macOS
+
+To configure your fancy zones you have to specify it them in the configuration using the ratios of the screen.
+
+<img alt="Zones example" src="./doc/zones.png" />
+
+The 0,0 point starts in the top-left corner. In order to obtain the result shown in the image you need 9 zones:
+
+```lua
+  zones = {
```

Comment:
I appreciate that the zones are highly flexible with this API (you can define "whatever you want"). However, I have to wonder whether 95% of the users want this kind of flexibility in their zone definitions?

Do they just want something like this?

```lua
zones = "3x3"
```

and then SkyRocket just does the math for them and makes them a grid?

I think that's what _I_ want as a user, what do you think?

Under the hood, you could convert "3x3" into the correct grids.

Reply:
This is a good idea for usability however it is restricting, maybe we can support both ways to define the grid.

In fact my current use case consists in two bigger zones and four smaller in a 4k screen with zoom at 100%

```lua
  zones = {
    {w=0.50,h=0.65,x=0.0 ,y=0},
    {w=0.50,h=0.65,x=0.50,y=0},
    {w=0.25,h=0.35,x=0.0, y=0.65},
    {w=0.25,h=0.35,x=0.25,y=0.65},
    {w=0.25,h=0.35,x=0.50,y=0.65},
    {w=0.25,h=0.35,x=0.75,y=0.65},
  }
```

I know that I am not an average user but it I think that the definition using ratios is simple enough for anybody to figure it out